docs/getting-started/cluster-setup.md

```diff
@@ -0,0 +1,232 @@
+# Setting Up a Cluster
```

Comment:
Hmm I was thinking there could be a separate topic somewhere here for Setting Up a Cluster on Jenkins
But if it makes more sense to have separate pages, I could do that too